src/test/unit/rx_spi_expresslrs_telemetry_unittest.cc

```diff
@@ -223,7 +223,7 @@ TEST(RxSpiExpressLrsTelemetryUnitTest, TestFlightMode)
 TEST(RxSpiExpressLrsTelemetryUnitTest, TestMspVersionRequest)
 { 
     uint8_t request[15] = {238,12,122,200,234,48,0,1,1,0,0,0,0,128, 0};
-    uint8_t response[12] = {200,10,123,234,200,48,3,1,0,1,44,42};
+    uint8_t response[12] = {200,10,123,234,200,48,3,1,0,1,45,255};
```

Comment:
Still tracking down why last byte needed to change from 42 -> 255 here. Second to last make sense, as it reflects the API version change